pkg/controller/nodes/controller.go

```diff
@@ -132,11 +133,12 @@ func (c *Controller) sendMarkedForTerminationEvent(dtc dtclient.Client, nodeIP s
 		return err
 	}
 
+	tenMinutesAgoInMillis := uint64(time.Now().Add(-10*time.Minute).UnixNano() / 1_000_000)
```

Comment:
The CircleCI pipeline uses Go 1.12 and this is a Go 1.13's number literal which makes compilation fail. Added a commit to upgrade Go.